### PR TITLE
Bring in proof-of-concept console renderer

### PR DESCRIPTION
**Pull request by paulcam206**

### CODE REVIEW

File: source/console/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Windows Console Renderer for Adaptive Cards
```
Comment (khouzam) [inline source/console/README.md]: I would suggest making moving it into a HACK or experimental folder to make sure that people aren't confused.

File: source/experimental/ConsoleApplication/ConsoleApplication/ConsoleApplication.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+{
+	DWORD dwRet = 0;
+	HANDLE hOut = GetStdHandle(STD_OUTPUT_HANDLE);
+
+	if (hOut != INVALID_HANDLE_VALUE)
+	{
+		GetConsoleMode(hOut, &dwRet);
+		SetConsoleMode(hOut, dwRet | ENABLE_VIRTUAL_TERMINAL_PROCESSING);
+	}
+
```
Comment (khouzam) [inline source/experimental/ConsoleApplication/ConsoleApplication/ConsoleApplication.cpp]: Tabs

File: source/experimental/ConsoleApplication/ConsoleApplication.sln [hunk 3]
```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27703.2042
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ConsoleApplication", "ConsoleApplication\ConsoleApplication.vcxproj", "{AA82D415-E4BA-48C7-85D8-6F675D70DED9}"
```
Comment (khouzam) [inline source/experimental/ConsoleApplication/ConsoleApplication.sln]: >ConsoleApplication [](start = 53, length = 18)

Perhaps this should be renamed to a slightly less generic name.

Comment (paulcam206) [reply source/experimental/ConsoleApplication/ConsoleApplication.sln]: Renamed the console application `adapt`.

**Pull request merged**